docs/networking/calico.md

```diff
@@ -55,37 +75,45 @@ To enable this mode in a cluster, add the following to the cluster spec:
     calico:
       crossSubnet: true
 ```
-In the case of AWS, EC2 instances have source/destination checks enabled by default.
-When you enable cross-subnet mode in kOps 1.19+, it is equivalent to:
+In the case of AWS, EC2 instances' ENIs have source/destination checks enabled by default.
+When you enable cross-subnet mode in kOps 1.19+, it is equivalent to either:
 ```yaml
   networking:
     calico:
       awsSrcDstCheck: Disable
+      encapsulationMode: ipip
```

Comment:
Maybe would be better to remove this for simplicity as it should be the default.
```suggestion
      encapsulationMode: ipip
```